menu.yaml

```diff
@@ -4,11 +4,21 @@
   icon: default.png
   index-patterns:
   - panels/json/git-index-pattern.json
+  - panels/json/git_areas_of_code-index-pattern.json
+  - panels/json/all_onion-index-pattern.json
   menu:
   - name: Overview
     panel: panels/json/git.json
   - name: Attraction/Retention
     panel: panels/json/git_demographics.json
+  - name: Areas of code
+    panel: panels/json/git_areas_of_code.json
+  - name: Community Structure Overall
```

Comment:
@alpgarcia told me they (he and @dicortazar ) want the onion panels under a new menu entry called "Community", not under "Git".

About the name of the sub-entry for those panels on the lateral menu, if they are going to be under "Community", they won't need that string in the name. I think `Structure Overall`, `Structure by projects` and `Structure by Organizations` should be ok (furthermore, the name is shorter).
Anyway, @lcanas told me yesterday he though the name will be "Overall", "By Projects" and "By Organizations", so that's another option but I think it could be a little bit confussing.